front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -147,14 +188,18 @@ const StcdmResults = ({
             isReadOnly
             hideAttribution
             showStdcmAssets
+            isFeasible={!hasConflicts}
             setMapCanvas={setMapCanvas}
-            pathGeometry={selectedSimulation.outputs?.pathProperties.geometry}
-            simulationPathSteps={selectedSimulation.outputs?.results.simulationPathSteps}
+            pathGeometry={
+              hasConflicts
+                ? pathProperties?.geometry
+                : selectedSimulation?.outputs?.pathProperties?.geometry
+            }
```

Comment:
When you'll have made the change for results.input/ouput

```suggestion
            pathGeometry={ selectedSimulation.outputs?.pathProperties.geometry}
```